Add content/pt-br/contributor-ladder (#662)
cncf/glossary: content/pt-br/contributor-ladder/_index.md

---
title: Escala de Contribuição
toc_hide: true
menu:
  main:
    weight: 10
    pre: <i class='fas fa-pen-square'></i>
---

> Olá! 👋 Obrigado pelo seu interesse em contribuir com o projeto CNCF Glossário Cloud Native. Seja contribuindo com novos termos, ajudando a localizar o glossário em seu idioma nativo ou ajudando outras pessoas a começar, há muitas maneiras de se tornar um membro ativo desta comunidade. Este documento descreve as diferentes funções do colaborador no projeto e as responsabilidades e privilégios que as acompanham.

Reviewer: I think would sound more natural if we invert the order of the sentences in the second paragraph:

```
Há muitas maneiras de se tornar um membro ativo desta comunidade - seja contribuindo com novos termos, ajudando a localizar o glossário em seu idioma nativo ou ajudando outras pessoas a começar a contribuir.
```
Something like this..what do you think?

## 1. Contribuidores

O glossário é para todos. Qualquer pessoa pode se tornar um contribuidor simplesmente contribuindo com o projeto.
É esperado que todos os contribuidores sigam o [código de conduta da CNCF](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).

Existem várias formas de você contribuir com o projeto, incluindo:

> - **Contribuidores de conteúdo**: qualquer pessoa que melhore os termos existentes ou contribua com novos,

Reviewer: I suggest to drop the comma at the end of each point - even though in the english version there is a comma there

Author: Hmm, I was [searching](https://ead.pucpr.br/blog/quando-usar-ponto-virgula), and looks like in Portuguese we can use the `;`, what do you think?

- **Contribuidores de localizações**: qualquer pessoa que ajude a localizar o conteúdo para outro idioma,
- **Voluntários**: qualquer pessoa que ajude os outros no Github, Slack ou qualquer membro da comunidade que precise de ajuda,
- **Embaixadores**: qualquer pessoa que ajude a divulgar o projeto e ajudar a a contribuir.

> Contribuidores podem ter múltiplas funções ou focar em uma única área. **Todas as contribuições são igualmente importantes** e ajudam no crescimento da comunidade. Para contribuições em conteúdo e localização, veja o [guia de contribuição](https://glossary.cncf.io/pt-br/contribute/) e o [guia de estilo](https://glossary.cncf.io/pt-br/style-guide/).

Reviewer: the 2 links are broken - would it be updated once we have them merged in `dev-pt`?

## 2. Aprovadores

Aprovadores são responsáveis por prover feedbacks e aprovar os PRs (pull requests). Um contribuidor ativo pode se tornar um aprovador (veja "Se tornando um aprovador"). O glossário possui dois tipos de aprovadores: (1) aprovadores para o glossário em inglês e (2) aprovadores para os times de localização.

É esperado que os aprovadores do glossário:

- Revisem os PRs em uma perspectiva técnica,
- Atribuiam as *issues* aos contribuidores e apliquem as *labels* corretamente,
- Ajudem os contribuidores com feedback e toda a ajuda que precisem.

Se um aprovador não estiver mais interessado ou não consegue realizar as atividades acima, é importante que comunique aos mantenedores para que seja formalizada a saída da função.

> ## Aprovadores do Glossário em Inglês

Reviewer: ```suggestion
### Aprovadores do Glossário em Inglês
```

as per https://glossary.cncf.io/contributor-ladder/#english-glossary-approvers and https://raw.githubusercontent.com/cncf/glossary/main/content/en/contributor-ladder/_index.md

Existem três tipos de aprovadores:

1) Aprovador com experiência técnica,
2) Aprovador com experiência em escrita,
3) Aprovador com proficiência nas duas habilidades acima.

> **Aprovadores com experiência técnica**: pessoas com muita experiência técnica podem ser aprovadores sem possui habilidades sólidas de escrita em inglês, entretanto, se aprovarem um PR pelo critério técnico, é preciso garantir que vai ser revisado por um aprovador com habilidades de escrita.

Reviewer: ```suggestion
**Aprovadores com experiência técnica**: pessoas com muita experiência técnica podem ser aprovadores sem possui habilidades sólidas de escrita em inglês, entretanto, se aprovarem um PR pelo critério técnico, é preciso garantir que vai ser revisado por um aprovador com habilidades de escrita (editor).
```

**Editores**: Os editores revisam os termos e garantem que eles sejam explicados em linguagem simples de acordo com o Guia de Estilo. Se um termo for muito editado, o editor deve solicitar a um aprovador técnico que o revise novamente para garantir que o significado não foi alterado.

### Aprovadores de localização

> O glossário possui também aprovadores de localização. Estes são aprovadores para um dos times de localização (times que traduzem o glossário). Aprovadores de localização apenas tem permissões para aprovar PRs para seu próprio time e também fazer o merge na branch de desenvolvimento dedicada ao seu time. Um aprovador de localização também pode se tornar um aprovador do glossário em inglês caso atenda os requisitos.

Reviewer: ```suggestion
O glossário possui também aprovadores de localização. Estes são aprovadores para um dos times de localização (times que traduzem o glossário). Aprovadores de localização apenas tem permissões para aprovar PRs para seu próprio time e também fazer o merge na branch de desenvolvimento dedicada ao seu time. Um aprovador de localização também pode se tornar um aprovador do glossário em inglês caso atenda aos requisitos.
```

### Se tornando um aprovador

Os candidatos a aprovador devem ter um histórico comprovado de envio de PRs de alta qualidade e de ajudar outras pessoas a obter deixar seus PRs prontos para o merge. Se o fuso horário permitir, eles também devem participar regularmente das [reuniões do Grupo de Trabalho do Glossário](https://www.cncf.io/calendar/).

> Para se tornar um aprovador, comece manifestando interesse aos mantenedores existentes. Os mantenedores existentes solicitarão que você demonstre as qualificações acima, contribuindo com PRs, fazendo revisões e realizando outras tarefas sob sua orientação. Depois de algum tempo trabalhando juntos, os mantenedores decidirão se concederão a você o status de aprovador. Esta decisão será baseada em seu nível demonstrado de proficiência e capacidade de resposta.

Reviewer: ```suggestion
Para se tornar um aprovador, comece manifestando interesse aos mantenedores existentes. Os mantenedores existentes solicitarão que você demonstre as qualificações acima, contribuindo com PRs, fazendo revisões e realizando outras tarefas sob sua orientação. Depois de algum tempo trabalhando juntos, os mantenedores decidirão se concederão a você o status de aprovador. Esta decisão será baseada em seu nível demonstrado de proficiência e responsividade.
```

## 3. Mantenedores

Mantenedores são aprovadores que também podem fazer o merge de PRs. Qualquer pessoa pode se tornar um mantenedor do glossário (veja "Se tornando um mantenedor"). Existem algumas responsabilidades esperadas dos mantenedores, incluindo:

- Ser um aprovador ativo e responsável,
- Ajudar a manter o repositório, incluindo configurações do site, permissões, templates de issues, workflows do Github e outras coisas,
- Acompanhar o slack do glossário e ajudar sempre que possível,
- Participar regularmente dos [encontros do Grupo de Trabalho do Glossário](https://www.cncf.io/calendar/)(sempre que for possível).

Se um mantenedor não estiver mais interessado ou não puder realizar as atividades acima, ele deve ser movido para o status de emérito.

### Se tornando um mantenedor

Os mantenedores devem ter um histórico comprovado de aprovação de sucesso e envio de PRs de alta qualidade. Se o fuso horário permitir, eles também devem participar regularmente das reuniões do Grupo de Trabalho do Glossário.

> Para se tornar um mantenedor, comece expressando interesse aos mantenedores existentes. Os mantenedores existentes solicitarão que você demonstre as qualificações acima, contribuindo com PRs, fazendo revisões e realizando outras tarefas sob sua orientação. Depois de algum tempo trabalhando juntos, os mantenedores decidirão se concederão o status de mantenedor. Esta decisão será baseada no nível demonstrado de proficiência e capacidade de resposta.

Reviewer: ```suggestion
Para se tornar um mantenedor, comece expressando interesse aos mantenedores existentes. Os mantenedores existentes solicitarão que você demonstre as qualificações acima, contribuindo com PRs, fazendo revisões e realizando outras tarefas sob sua orientação. Depois de algum tempo trabalhando juntos, os mantenedores decidirão se concederão o status de mantenedor. Esta decisão será baseada no nível demonstrado de proficiência e responsividade.
```

> ## 4. Community Managers

Reviewer: Not sure if here would be nicer `Mantenedores de comunidade`?

Author: Hmm, it's related to the GitHub repository, I'm not sure If we can use `mantenedores de comunidade`  what you think @brunoguidone?

Reviewer: Sounds good! ^
